social-network/src/MissedArticles.js

```diff
@@ -0,0 +1,39 @@
+// missedArticles.js
+import React from 'react';
+import missedArticlesJson from './missed-articles.json';
+
+class missedArticlesComp extends React.Component {
+
+    render() {
+        const articles = [];
+        for (var i=0; i < missedArticlesJson.length; i++) {
```

Comment:
This loop isn't needed, `missedArticlesJson` can be referenced directly instead.